lib/routes/account.js

```diff
@@ -646,6 +734,7 @@ module.exports = function (
             response.verificationMethod = 'email'
             response.verificationReason = 'login'
           }
+          recordSecurityEvent() // don't block response recording db event
```

Comment:
Oh, just meant to put it in this format

```
         customs.check(request, email, 'accountLogin')
           .then(readEmailRecord)
           .then(checkSecurityHistory)
           .then(checkNumberOfActiveSessions)
           .then(createSessionToken)
           .then(createKeyFetchToken)
           .then(emitSyncLoginEvent)
           .then(sendVerifyAccountEmail)
           .then(sendNewDeviceLoginNotification)
           .then(sendVerifyLoginEmail)
           .then(recordSecurityEvent)
           .then(createResponse)
           .done(reply, reply)
```

where `recordSecurityEvent` is before `createResponse`